packages/eslint-config/test/ts.spec.js

```diff
@@ -19,50 +19,5 @@ it('checks that the rules are working', async () => {
 it('checks for rule deprecations', async () => {
   const [result] = await getTSEslint().lintFiles([fileName]);
 
-  expect(result.usedDeprecatedRules).toMatchInlineSnapshot(`
-    [
-      {
-        "replacedBy": [
-          "@stylistic/ts/padding-line-between-statements",
-        ],
-        "ruleId": "@typescript-eslint/padding-line-between-statements",
-      },
-      {
-        "replacedBy": [],
-        "ruleId": "global-require",
-      },
-      {
-        "replacedBy": [
-          "padding-line-between-statements",
-        ],
-        "ruleId": "lines-around-directive",
-      },
-      {
-        "replacedBy": [],
-        "ruleId": "no-buffer-constructor",
-      },
-      {
-        "replacedBy": [
-          "no-object-constructor",
-        ],
-        "ruleId": "no-new-object",
-      },
-      {
-        "replacedBy": [],
-        "ruleId": "no-new-require",
-      },
-      {
-        "replacedBy": [],
-        "ruleId": "no-path-concat",
-      },
-      {
-        "replacedBy": [],
-        "ruleId": "no-return-await",
-      },
```

Comment:
> This rule was deprecated in ESLint v8.46.0 with no replacement. The original intent of this rule no longer applies due to the fact JavaScript now handles native Promises differently. It can now be slower to remove await rather than keeping it. More technical information can be found in [this V8 blog entry](https://v8.dev/blog/fast-async).